src/cli/mod.rs

```diff
@@ -100,12 +128,25 @@ struct ReplArgs {
     #[clap(long, value_parser)]
     load: Option<PathBuf>,
 
+    /// Config file (higher precedence than env vars and lower than CLI args)
```

Comment:
I think you mean the config file has the lowest precedence, not the highest.

Reply:
Done